apps/mobile/src/features/balances/stacks/stacks-balance.tsx

```diff
@@ -10,7 +10,11 @@ interface StacksTokenBalanceProps extends PressableProps {
   availableBalance: Money;
   fiatBalance: Money;
 }
-export function StacksTokenBalance({ availableBalance, fiatBalance }: StacksTokenBalanceProps) {
+export function StacksTokenBalance({
+  availableBalance,
+  fiatBalance,
+  ...rest
```

Comment:
I don't think we need this `...rest` as we don't need `onPress` and if we do maybe just pass that prop instead?

Reply:
Its type signature initially extended PressableProps, so I think it's only fair that it passes down what's defined, and it enabled expected behavior. It's probably worth revisiting these components both for polishing type signatures, but also improving the naming (these aren't exactly Balance components), and maybe trimming down the variations to just 1-2. But that's way out of scope for this PR.